src/res/svg/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Platform } from 'react-native';
+
+import PlaceholderIcon from './ic_placeholder_24dp'
+import CrosshairsIcon from './ic_crosshairs_24dp'
+import MegaphoneIcon from './ic_megaphone_24dp'
+import TwitterIcon from './ic_twitter_24dp'
+import CloseIcon from './ic_close_24dp'
+import ShareDroidIcon from './ic_share_droid_24dp'
+import ShareiOSIcon from './ic_share_ios_24dp'
+import OverflowiOSIcon from './ic_overflow_ios_24dp'
+import OverflowDroidIcon from './ic_overflow_droid_24dp'
+
+Platform.OS === 'android'
+const Icon = props => {
+  if (props.icon === 'crosshairs') {
+    return <CrosshairsIcon {...props} />;
+
+  } else if (props.icon === 'megaphone') {
+    return <MegaphoneIcon {...props} />;
+
+  } else if (props.icon === 'twitter') {
+    return <TwitterIcon {...props} />;
+
+  } else if (props.icon === 'close') {
+    return <CloseIcon {...props} />;
+
+  } else if (props.icon === 'share' && Platform.OS === 'android') {
+    return <ShareDroidIcon {...props} />;
+
+  } else if (props.icon === 'share' && Platform.OS === 'ios') {
+    return <ShareiOSIcon {...props} />;
+
+  } else if (props.icon === 'overflow' && Platform.OS === 'android') {
+    return <OverflowDroidIcon {...props} />;
+
+  } else if (props.icon === 'overflow' && Platform.OS === 'ios') {
+    return <OverflowiOSIcon {...props} />;
+  
+  } else {
+    return <PlaceholderIcon {...props} />;
+  }
```

Comment:
I think the below suggestion makes the code more readable and scale better as the number of different icons increases.
```suggestion
const isAndroid = R.equals('android');
const icons = {
  crosshairs: R.always(CrosshairsIcon),
  ...,
  overflow: platform => isAndroid(platform) ? OverflowDroidIcon : OverflowiOSIcon,
};
const matchedIcon = icons[props.icon];
const ValidatedIcon = matchedIcon ? matchedIcon(Platform.OS) : PlaceholderIcon;
return <ValidatedIcon {...props} />;
```